### PR TITLE
Add MBart translation component

### PR DESCRIPTION
**Pull request by heinpa**
Implement the non-Qanary implementation for MBart (https://github.com/dice-group/LFQA/blob/main/naive-eamt/component/mbart_mt.py) as a Qanary component.

### CODE REVIEW

File: qanary-component-MT-Python-MBart/app/mt_mbart_nlp.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+mt_mbart_nlp_bp = Blueprint("mt_mbart_nlp_bp", __name__, template_folder="templates")
+
+SERVICE_NAME_COMPONENT = os.environ["SERVICE_NAME_COMPONENT"]
+supported_langs = ["en"] # TODO: check supported languages for LibreTranslate
+langid.set_languages(supported_langs)
+# TODO: check code mapping
+
+model = MBartForConditionalGeneration.from_pretrained("facebook/mbart-large-50-many-to-many-mmt")
+tokenizer = MBart50TokenizerFast.from_pretrained("facebook/mbart-large-50-many-to-many-mmt")
+lang_code_map = {
```
Comment (Perevalov) [inline qanary-component-MT-Python-MBart/app/mt_mbart_nlp.py:21]: What does the `en_XX` mean? Is it expected to type it like this?

File: qanary-component-MT-Python-MBart/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+tokenizers
+tomli
+torch
+tqdm
+transformers
+typing_extensions
+urllib3
+websocket-client
+Werkzeug
+zipp
```
Comment (Perevalov) [inline qanary-component-MT-Python-MBart/requirements.txt:93]: Are you sure that such a long list of requirements is needed? I suggest to use `pipreqs` for generating it

Comment (heinpa) [reply qanary-component-MT-Python-MBart/requirements.txt:93]: I will try to reduce the list of requirements

File: qanary-component-MT-Python-MBart/app/mt_mbart_nlp.py (hunk 1 above)
Comment (Perevalov) [reply qanary-component-MT-Python-MBart/app/mt_mbart_nlp.py:21]: Ok, fine.

**Comment by Perevalov** [comment]
@heinpa While building and running the component it showed an error that the `torch` library is missing. Seems, that `pipreqs` doesn't cover all the dependencies.

- [ ] Add `torch` to the `requirements.txt`

During the first start, the download of the model from huggingface is done. It takes some time, however, it's obviously more than a default `timeout` of `gunicorn`. It makes sense to add these lines too `boot.sh` for doing the download in advance:

```bash
echo Downloading the model
python -c 'from transformers import MBartForConditionalGeneration, MBart50TokenizerFast; model = MBartForConditionalGeneration.from_pretrained("facebook/mbart-large-50-many-to-many-mmt"); tokenizer = MBart50TokenizerFast.from_pretrained("facebook/mbart-large-50-many-to-many-mmt")'
echo Downloading the model finished
```

- [ ] Add downloading of the model prior to `gunicorn` start

**Comment by Perevalov** [comment]
I tested it locally, works.

**Pull request merged**